### PR TITLE
Collectors: Make icollect take init files too

### PR DESCRIPTION
**Pull request by Adrianzatreanu**

### CODE REVIEW

**Comment by hypothesist** [comment]
You mean `icollect_bears`, right? There are other icollects too :P

**Comment by Adrianzatreanu** [comment]
i fear it will be over 50 chars :(

**Comment by Adrianzatreanu** [comment]
changed

File: coalib/collecting/Collectors.py [hunk 1]
```diff
@@ -117,7 +117,7 @@ def icollect_bears(bear_dirs, bear_globs, kinds, log_printer):
         # Since we get a real directory here and since we
         # pass this later to iglob, we need to escape this.
         bear_dir = glob_escape(bear_dir)
-        for bear_glob in bear_globs:
+        for bear_glob in bear_globs + ["__init__"]:
```
Comment (sils1297) [inline coalib/collecting/Collectors.py]: nah I think that would *always* collect a bear that is in `init`

**Comment by sils1297** [comment]
Hey, I'm pretty sure that this is buggy! Marking WIP.

**Pull request closed without merging**